### PR TITLE
Changed the required permission for VM creation

### PR DESCRIPTION
**Pull request by zefciu**
You need core access to create a VM via API.

### CODE REVIEW

File: src/ralph/ui/tests/global_utils.py [hunk 1]
```diff
@@ -52,6 +62,14 @@ def setUp(self):
         cache.delete("api_user_accesses")
         super(UserTestCase, self).setUp()
 
+
+    def add_perms(self, perms):
```
Comment (kula1922) [inline src/ralph/ui/tests/global_utils.py]: docstring

File: src/ralph/ui/tests/global_utils.py [hunk 2]
```diff
@@ -52,6 +62,14 @@ def setUp(self):
...
+        user_profile = Profile.objects.get(user=self.user)
+        for perm in perms:
+            BoundPerm(profile=user_profile, perm=perm).save()
+        user_profile.save()
```
Comment (kula1922) [inline src/ralph/ui/tests/global_utils.py]: why user_profile.save()? There is no any change.

File: src/ralph/ui/tests/global_utils.py [hunk 3]
```diff
@@ -52,6 +62,14 @@ def setUp(self):
...
+        self.user.save()
```
Comment (kula1922) [inline src/ralph/ui/tests/global_utils.py]: dito

File: src/ralph/ui/tests/global_utils.py (hunk 1 above)
Comment (zefciu) [inline src/ralph/ui/tests/global_utils.py]: Fixed

File: src/ralph/ui/tests/global_utils.py [hunk 4]
```diff
@@ -41,8 +41,18 @@ def login_as_su(username='ralph', password='ralph', email='ralph@ralph.local',
 class UserTestCase(TestCase):
     """A test case that creates a user and allows to test API with it."""
 
+    is_staff = True
+    is_superuser = True
+
     def setUp(self):
-        self.user = create_user('api_user', 'test@mail.local', 'password')
+        self.user = create_user(
```
Comment (ar4s) [inline src/ralph/ui/tests/global_utils.py]: maybe you should use ``UserFactory`` from ``ralph.ui.tests.global_utils``?

**Comment by kula1922** [comment]
:ok:

**Comment by ar4s** [comment]
:ok:

**Pull request merged**